<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OCC::WifiListManager</class>
 <widget class="QDialog" name="OCC::WifiListManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>420</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>420</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Nextcloud Wi-Fi lists</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QGroupBox" name="listModeGroup">
       <property name="title">
        <string>List Mode</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QRadioButton" name="whitelistCheckbox">
          <property name="text">
           <string>Whitelisting</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="blacklistCheckbox">
          <property name="text">
           <string>Blacklisting</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>103</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="WiFiListsVertical">
     <item>
      <widget class="QGroupBox" name="SSIDgroup">
       <property name="title">
        <string>SSIDs</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QListWidget" name="ssidList">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="showDropIndicator" stdset="0">
           <bool>false</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::ExtendedSelection</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="ListEditEntries">
          <item>
           <widget class="QLineEdit" name="ssidEntryEdit">
            <property name="minimumSize">
             <size>
              <width>140</width>
              <height>0</height>
             </size>
            </property>
            <property name="placeholderText">
             <string>Enter SSID here</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="addToListBtn">
            <property name="text">
             <string>Add entry</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="deleteFromListBtn">
            <property name="text">
             <string>Delete entry</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="saveExitBtn">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelExitBtn">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>whitelistCheckbox</tabstop>
  <tabstop>blacklistCheckbox</tabstop>
  <tabstop>ssidEntryEdit</tabstop>
  <tabstop>addToListBtn</tabstop>
  <tabstop>deleteFromListBtn</tabstop>
  <tabstop>ssidList</tabstop>
  <tabstop>saveExitBtn</tabstop>
  <tabstop>cancelExitBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
